openspec/specs/github-hierarchy-cache/spec.md

```
@@ -0,0 +1,33 @@
+# github-hierarchy-cache Specification
+
+## Purpose
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)